core/src/com/testing/TestingApp.java

```diff
@@ -30,4 +41,47 @@ public void render() {
     public void dispose() {
 
     }
+
+    /**
+     * generate all fonts to be used in the game
+     */
+    protected void generateFonts(){
```

Comment:
This is not a change request. Maybe this should be moved to some `FontManager` class?

Reply:
Good idea, I'll open a new issue for it and work on it next.